src/BenchmarksServer/Startup.cs

```diff
@@ -289,7 +273,7 @@ private static void DeleteDir(string path)
         private static Process StartProcess(string hostname, string benchmarksRepo, ServerJob job)
         {
             var filename = "dotnet";
-            var arguments = $"run -c Release -- --scenarios {job.Scenario} --server {job.WebHost} " +
+            var arguments = $"run -c Release -f netcoreapp1.1 -- --scenarios {job.Scenario} --server {job.WebHost} " +
```

Comment:
I'm confused.  Is it required to add `netcoreapp1.1` to `dotnet run` as part of the MSBuild conversion?